doc/source/development/contributing_codebase.rst

```diff
@@ -807,8 +804,45 @@ across the number of specified cores or ``auto`` to utilize all the available co
    # Utilizes all available cores
    pytest -n auto pandas
 
-This can significantly reduce the time it takes to locally run tests before
-submitting a pull request.
+If you'd like to speed things along further a more advanced use of this
+command would look like this
+
+.. code-block:: bash
+
+    pytest pandas --skip-slow --skip-network --skip-db -m "not single_cpu" -n 4 -r sxX
+
+In addition to the multithreaded performance increase this improves test
+speed by skipping some tests:
+
+- skip-slow: any test that takes long (think seconds rather than milliseconds)
+- skip-network: tests that require network connectivity
+- skip-db: tests that require a database
+
+The -r flag will display a short summary info (see `pytest documentation <https://docs.pytest.org/en/4.6.x/usage.html#detailed-summary-report>`_)
```

Comment:
Maybe a side discussion, but we _could_ also consider adding a default selection of `-r` flags to the pytest configuration in pyproject.toml ("addopts")

(eg dask adds `-rsxfE`: https://github.com/dask/dask/blob/5ba240b9aa4711ddbcea2accfe7139552cab0208/setup.cfg#L56)